src/init.c

```diff
@@ -1048,7 +1048,7 @@ _dispatch_bug_kevent_vanished(dispatch_unote_t du)
 			"{ %p[%s], ident: %" PRIdPTR " / 0x%" PRIxPTR ", handler: %p }",
 			dux_type(du._du)->dst_kind, dou._dq,
 			dou._dq->dq_label ? dou._dq->dq_label : "<unknown>",
-			du._du->du_ident, du._du->du_ident, func);
+			(intptr_t)du._du->du_ident, (intptr_t)du._du->du_ident, func);
```

Comment:
If we're going to cast here, we might as well choose `uintmax_t` (in the alternative, I had suggested on the other pr defining some custom format strings because the type of `dispatch_unote_ident_t` is platform defined, but casting is a reasonable alternative here.)